Jellyfin.Server/ServerSetupApp/SetupServer.cs

```diff
@@ -80,6 +80,37 @@ public SetupServer(
     /// </summary>
     public bool IsAlive { get; internal set; }
 
+    private static void ApplyDefaultHostConfiguration(IConfigurationBuilder hostConfigBuilder, string[]? args)
```

Comment:
I think this could use some documentation, this was extracted from within `CreateDefaultBuilder` correct?